src/server/room-controller.spec.js

```diff
@@ -29,13 +29,20 @@ describe('room-controller', () => {
           return Promise.resolve(room);
         }
         return Promise.resolve();
+      }),
+      isRoomMemberOrOwner: sandbox.stub().callsFake(roomId => {
```

Comment:
The `.callsFake(...` part handles a very specific scenario used in one `describe` block, so I think it would be better to move that into the `beforeEach` of that block, leaving here only `isRoomMemberOrOwner: sandbox.stub()`, so other tests can do other things with it.

BTW: I did that already for the function above (`getRoomById`) in my branch.

Reply:
Fixed